src/main/java/com/sap/oss/phosphor/fosstars/data/github/VulnerabilitiesFromNvd.java

```diff
@@ -55,6 +55,7 @@ public VulnerabilitiesFromNvd(GitHubDataFetcher fetcher, NVD nvd) {
       vulnerabilities.add(Vulnerability.Builder.from(entry).make());
     }
 
+    logger.info("Got ... {}", vulnerabilities);
```

Comment:
It maybe quite a long list of issues sometimes.

```suggestion
    logger.info("Found {} {}", vulnerabilities.size(), vulnerabilities.size() == 1 ? "vulnerability" : "vulnerabilities");
```